tests/poly/lower_poly.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: heir-opt --lower-poly %s > %t
+// RUN: FileCheck %s < %t
+
+#cycl_2048 = #poly.polynomial<1 + x**1024>
+#ring = #poly.ring<cmod=4294967296, ideal=#cycl_2048>
+module {
+  func.func @test_lower_from_coeffs() {
+    %c0 = arith.constant 0 : index
+    // 2 + 2x + 5x^2
+    %coeffs = "arith.constant"() <{value = dense<[2, 2, 5]> : tensor<3xi32>}> : () -> tensor<3xi32>
```

Comment:
Why the generic form of arith.constant here?

Reply:
My LSP recommended it as a replacement for the sequence of ops I had there before (arith.constant for each individual value + tensor.from_elements). I wasn't sure what the right syntax was for an `arith.constant` with an attribute, since there's no argument there. Will simplify it.